### DISCUSSION TITLE
[Flutter]Execution failed for task ':app:processDebugMainManifest'.

### DISCUSSION THREAD

**Discussion by akth101** [community/community#70085] [Programming Help]
### Body

I got this error when I click the run button to test my code.
I`m developing with Mac and using android studio.
I tried so many methods from googoling, But none of them can`t resolve my problem.
What should I have to do with this error?

Error message:
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processDebugMainManifest'.
> Multiple build operations failed.
      java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/3f000b05d3cf213f3004ae1670ab03a4/results.bin
      java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/a4ddf7816da8fe90c31122ba833bd9de/results.bin
      java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/6109d0b0cb22b34b3af88f4a1b162c03/results.bin
      java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/ddc77d84a2427d669fc9545cef23fa36/results.bin
      java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/d302745e954a857085547d6296884920/results.bin
      java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/6d26f918b831a182a23f69e63385a70f/results.bin
      java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/9156205b9df430067e7882b9d72e0538/results.bin
      java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/9222beed16560c7b4837893555d572af/results.bin
      java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/9e30150b9d96e94744f0b9c18335b952/results.bin
      java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/49505ced950b0e261a1c793e28c2bc6d/results.bin
      ...and 31 more failures.
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/3f000b05d3cf213f3004ae1670ab03a4/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/a4ddf7816da8fe90c31122ba833bd9de/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/6109d0b0cb22b34b3af88f4a1b162c03/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/ddc77d84a2427d669fc9545cef23fa36/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/d302745e954a857085547d6296884920/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/6d26f918b831a182a23f69e63385a70f/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/9156205b9df430067e7882b9d72e0538/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/9222beed16560c7b4837893555d572af/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/9e30150b9d96e94744f0b9c18335b952/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/49505ced950b0e261a1c793e28c2bc6d/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/830873f5d263e2a17dc30404c91ef62e/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/92f520a6c0f0dec86bf0b267044e8d8d/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/9d6990d9abd3c351763e0c181c1a65c2/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/7e76fab88cf14cb4d97ecffa278e49d1/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/c852596f875bda11414d6043a565364d/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/69c5051fbb17075a2ab4feb441a104a6/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/95485a9cc578832748cb5674b58b3609/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/b589a790aada9d711e36e6bd021a1bb4/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/f0aba97deccad48111d075606e8fe9bb/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/00a24eed1fa39224eb7a8aa7f0d0b76b/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/9d6990d9abd3c351763e0c181c1a65c2/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/9cb8e0441e9a6c7581f6d66e725703f8/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/c852596f875bda11414d6043a565364d/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/ad2de6ce07120a3a7a18bee05f9da3e0/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/6d46d7241514740164a06753290ce719/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/ae5e8ddcf1ebc781ba349bd404292eaa/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/1c3398abf418663c78f2408cc37f8b80/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/00a24eed1fa39224eb7a8aa7f0d0b76b/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/89017a3548906e18289c48f634a99982/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/8dac7946bb55616c48ee163412aed335/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/194093ef58a74cb7112f96742c16a58b/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/53d56a5800db161f4dc87d4aab753bdb/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/57e6837cd19349f12c4c4599d5fd50bf/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/92f86096b732c1d85052e9c384260640/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/194093ef58a74cb7112f96742c16a58b/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/7e76fab88cf14cb4d97ecffa278e49d1/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/c852596f875bda11414d6043a565364d/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/0a956395a35c6cccce3dadcadfc9f236/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/c852596f875bda11414d6043a565364d/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/c852596f875bda11414d6043a565364d/results.bin
   > java.nio.file.NoSuchFileException: /Users/koseongjun/.gradle/caches/transforms-3/c852596f875bda11414d6043a565364d/results.bin

------------------------------------------------------------------------------------------------

my build.gradle file:

buildscript {
    ext.kotlin_version = '1.7.10'
    repositories {
        google()
        mavenCentral()
    }

    dependencies {
        classpath 'com.android.tools.build:gradle:7.3.0'
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
    }
}

allprojects {
    repositories {
        google()
        mavenCentral()
    }
}

rootProject.buildDir = '../build'
subprojects {
    project.buildDir = "${rootProject.buildDir}/${project.name}"
}
subprojects {
    project.evaluationDependsOn(':app')
}

tasks.register("clean", Delete) {
    delete rootProject.buildDir
}

------------------------------------------------------------------------------------------------

flutter doctor -v results:
[✓] Flutter (Channel stable, 3.13.7, on macOS 13.2 22D49 darwin-arm64, locale ko-KR)
    • Flutter version 3.13.7 on channel stable at /Users/koseongjun/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 2f708eb839 (5일 전), 2023-10-09 09:58:08 -0500
    • Engine revision a794cf2681
    • Dart version 3.1.3
    • DevTools version 2.25.0

[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at /Users/koseongjun/Library/Android/sdk
    • Platform android-34, build-tools 34.0.0
    • Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 17.0.6+0-17.0.6b829.9-10027231)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 14.3.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 14E300c
    • CocoaPods version 1.12.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2022.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.6+0-17.0.6b829.9-10027231)

[✓] VS Code (version 1.70.2)
    • VS Code at /Users/koseongjun/Downloads/Visual Studio Code.app/Contents
    • Flutter extension version 3.60.0

[✓] Connected device (3 available)
    • sdk gphone64 arm64 (mobile) • emulator-5554 • android-arm64  • Android 14 (API 34) (emulator)
    • macOS (desktop)             • macos         • darwin-arm64   • macOS 13.2 22D49 darwin-arm64
    • Chrome (web)                • chrome        • web-javascript • Google Chrome 118.0.5993.70

[✓] Network resources
    • All expected network resources are available.

• No issues found!

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by heysem-useinsider** [comment]
Hi,
This error is probably a result of a corrupted Gradle cache, try doing the following in order, after each step try to rebuild the project and see if it's working, if not go to the next step:

In Android Studio:

1. Select `File` > `Sync Project with Gradle Files"
2. Select `Build` > `Clean Project`.
3. Select `File` > `Invalidate Caches / Restart...` > `Invalidate and Restart`.
4. Select `View` > `Tool Windows` > `Terminal`, and run this command `rm -rf ~/.gradle/caches/`.

If those steps don't work for you, try also checking your Android SDK. Make sure it's installed and referenced in Android Study settings.
